multi-s3-output/out_multi_s3.go

```diff
@@ -239,18 +240,17 @@ func ArchiveLog(data []byte, dst io.Writer) int {
 			timestamp = time.Now()
 		}
```

Comment:
Getting timestamp from timestamp record field looks like a good candidate to factoring out.

Reply:
done